packages/nimble-components/src/tree-item/styles.ts

```diff
@@ -139,12 +144,12 @@ export const styles = css`
     }
     .start {
         display: flex;
-        fill: currentcolor;
         margin-inline-start: ${iconSize};
         margin-inline-end: ${iconSize};
     }
 
     slot[name='start']::slotted(*) {
+        ${iconColor.cssCustomProperty}: currentcolor;
```

Comment:
Sorry, should this be `inherit` as well?

Reply:
No, `inherit` here wouldn't work since there isn't an `iconColor` to inherit from. We want to set the `iconColor` to the `color` used on the parent.

As a note, we do this in a few different places: menu-item, anchor-menu-item, and text-field